### PR TITLE
[DINGO-722] Remove IP checks

### PR DESCRIPTION
**Pull request by JeanMarcGoepfert**
IP checks are causing false positives for some users, so we've decided to remove them from ZAS.

Also installs required version of bundler in travis before_install hook to resolve failing builds.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,6 +15,8 @@ matrix:
   include:
     - rvm: 2.4.0
       env: COMMAND=rubocop
+before_install:
+  - gem install bundler:1.17.3
```
Comment (fvzzy) [inline .travis.yml]: Perhaps grab the bundler version from the `Gemfile.lock`, e.g.  `gem install bundler:"$(grep -A 1 "BUNDLED WITH" Gemfile.lock | tail -n 1)"`

File: .travis.yml [hunk 2]
```diff
@@ -15,6 +15,8 @@ matrix:
   include:
     - rvm: 2.4.0
       env: COMMAND=rubocop
+before_install:
```
Comment (mtsolakiszen) [inline .travis.yml]: Why was this not needed before but needed now?

File: .travis.yml (hunk 1 above)
Comment (JeanMarcGoepfert) [reply .travis.yml]: Removed this, was sorted in https://github.com/zendesk/zendesk_apps_support/pull/269

File: .travis.yml (hunk 2 above)
Comment (JeanMarcGoepfert) [reply .travis.yml]: A change in travis I guess. I've removed this as it's sorted in https://github.com/zendesk/zendesk_apps_support/pull/269

**Comment by token-cjg** [comment]
@JeanMarcGoepfert what are the risks of this change?

**Pull request merged**